docs/hub/datasets-viewer-embed.md

```diff
@@ -55,3 +55,18 @@ It will open a modal with the iframe code that you can copy and paste into your
 <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/dataset-viewer-embed-search-button-modal.png"/>
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/dataset-viewer-embed-search-button-modal-dark.png"/>
 </div>
+
+## Examples
+
+The embedded dataset viewer is used in multiple Machine Learning tools and platforms to display datasets. Here are a few examples:
+
+- [ZenML](https://www.zenml.io/blog/embedding-huggingface-datasets-visualizations-with-zenml)
+- [Metaflow + Outerbounds](https://www.linkedin.com/posts/eddie-mattia_the-team-at-hugging-facerecently-released-activity-7219416449084272641-swIu)
+- [AutoTrain](https://x.com/abhi1thakur/status/1813892464144798171)
```

Comment:
for some of them you could add a screenshot or video/GIF of their UI

and mention hf username of creator / link to hf profile 👍  (potentially add a widget to follow them directly from here? or badge from https://huggingface.co/datasets/huggingface/badges – just an idea though)

Reply:
yes, good ideas. Working on it

Reply:
PR: https://github.com/huggingface/hub-docs/pull/1344